src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -124,8 +125,15 @@ const ReportActionItemFragment = (props) => {
                         <Text
                             fontSize={variables.fontSizeSmall}
                             color={themeColors.textSupporting}
+                            style={[styles.alignItemsBaseline, editedLabelStyles]}
                         >
-                            {` ${props.translate('reportActionCompose.edited')}`}
+                            <Text
+                                selectable={false}
+                                style={[styles.w1, styles.userSelectNone]}
```

Comment:
Because we didn't set a line height for this `Text`, it caused a regression [here](https://github.com/Expensify/App/issues/19605).